tools/upgrade-contract-calibnet.sh

```diff
@@ -0,0 +1,62 @@
+#! /bin/bash
+# upgrade-contract-calibnet upgrades proxy at $PROXY_ADDRESS to a new deployment of the implementation 
+# of the contract at $IMPLEMENTATION_PATH (i.e. src/Payments.sol:Payments)
+# Assumption: KEYSTORE, PASSWORD, RPC_URL env vars are set to an appropriate eth keystore path and password
+# and to a valid RPC_URL for the calibnet.
+# Assumption: forge, cast, jq are in the PATH
+#
+echo "Upgrading Payments contract on calibnet"
```

Comment:
As with deploy we should move to one upgrade script across all networks -- just like we have for transfer and get owner.